ReleaseNotes.md

```diff
@@ -1,3 +1,6 @@
+# 3.1.0 (13 August 2022)
```

Comment:
No need to update this manually, I use https://github.com/StefH/GitHubReleaseNotes to generate this file.

Reply:
Reverted manual changes to ReleaseNotes.md